eclass/elisp-common.eclass

```diff
@@ -457,7 +457,7 @@ elisp-site-regen() {
 		# was actually no change.
 		# A case is a remerge where we have doubled output.
 		rm -f "${T}"/site-gentoo.el
-		eend
+		eend $?
```

Comment:
Good catch. Looking at it, I see that the eclass has some instances of `eend $? || die` as well.
So, could to take the eclass out of the PR? I'd prefer to fix all eend commands in one go.